src/WebJobs.Extensions.DurableTask/NoOpDisposable.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// A singleton disposable that does nothing when disposed.
+    /// From: https://github.com/StephenCleary/Disposables
+    /// </summary>
+    public sealed class NoOpDisposable : IDisposable
+#if NETSTANDARD2_1
+        , IAsyncDisposable
+#endif
+    {
+        private NoOpDisposable()
+        {
+        }
+
+#if NETSTANDARD2_1
+        /// <summary>
+        /// Does nothing.
+        /// </summary>
+        public ValueTask DisposeAsync() => new ValueTask();
+#endif
```

Comment:
Since we don't target .NET Standard 2.1 (and don't plan to), let's remove all the `IAsyncDisposable` stuff.